src/components/ComingSoon/ComingSoon.js

```diff
@@ -0,0 +1,48 @@
+import Button from '../Button/Button.vue';
+import Heading from '../Heading/Heading.vue';
+import TextParagraph from '../TextParagraph/TextParagraph.vue';
+import comingSoonImage from '../../assets/img_wip.svg';
```

Comment:
(nit) please rename this with PascalCase, as we do for Components: `ComingSoonImage`